src/packages/@workspaces/features/rest-explorer/components/request-name/RequestName.svelte

```diff
@@ -1,8 +1,9 @@
 <script lang="ts">
   import Input from "$lib/components/inputs/Input.svelte";
+  import type { UpdateRequestNameType } from "@workspaces/shared/type";
 
-  export let name;
-  export let onUpdateRequestName;
+  export let name: string;
```

Comment:
Comments missing everywhere for props and other places.

Reply:
fixed